test/go.mod

```diff
@@ -2,14 +2,16 @@ module github.com/metal3-io/cluster-api-provider-metal3/test
 
 go 1.23.7
 
+toolchain go1.24.1
```

Comment:
This is dependabot-core bug, since since this PR is about IRSO, I will manually fix it, so we can get this in.

/assign

Reply:
Fixed